src/main/webapp/dev/mashup.jsp

```diff
@@ -2,354 +2,601 @@
 <%@ page import="teammates.common.util.StringHelper"%>
 <%@ page import="teammates.logic.core.FeedbackQuestionsLogic"%>
 <%@ page import="teammates.logic.api.Logic"%>
-<%@ page import="teammates.common.datatransfer.StudentAttributes" %>
+<%@ page import="teammates.common.datatransfer.StudentAttributes"%>
```

Comment:
Remove unrelated changes due to indentation from the changeset. The pull request should only contain the required changes for the issue to be completed.